src/Symfony/Component/Serializer/Mapping/AttributeMetadata.php

```diff
@@ -127,17 +127,45 @@ public function getMaxDepth(): ?int
     /**
      * {@inheritdoc}
      */
-    public function setSerializedName(string $serializedName = null)
+    public function setSerializedNames(array $serializedNames): void
     {
-        $this->serializedName = $serializedName;
+        $this->serializedNames = $serializedNames;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function getSerializedName(): ?string
+    public function setSerializedName(string $serializedName = null, array $groups = [])
```

Comment:
Adding an argument is a BC break. We should add the argument as comment and read it with func_get_arg
See existing similar layer in the codebase for inspiration.